model.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, String, Float, Time
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+class LondonBikes(Base):
+    __tablename__ = 'bikes'
+
+    id = Column(String(16), primary_key=True, unique=True)
```

Comment:
If you keep the id as the primary key, you will not be able to save the historical availability of bikes in this table. How about we make a separate auto-increment primary key, and keep (id, time) unique?

Reply:
I am not exactly sure if time can be unique. After checking the API response, I found out that the times are same for few bike points.

Reply:
Absolutely, but a composite key of the columns (bikpoint_id and time) should be unique right? Also, this time should be the 'modified' property of the object containing the key 'NbBikes'.

Reply:
Okay got it.